Extension/lib/webrequest.js

```diff
@@ -250,6 +250,9 @@
 
             var options = null;
             if (!isEdgeBrowser) {
+                /**
+                 * In Edge browser: If we pass frameId in tabs.sendMessage then message aren't delivered to content-script

```

Comment:
No, we shouldn't. I think it will be fixed when Edge supports messaging to the specific frame. Now it doesn't.